### ISSUE TITLE
Nullish Colaescing Operator ?? causes an error

### ISSUE THREAD

**Issue by xavi-mat** [HookyQR/VSCodeMinify#68]
It seems that Minify can't manage the nullish coalescing operator.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

I'm trying to minimize this:
```
const a = document.nonExistent ?? "Not found";
```

And get:
> Minify failed: Unexpected token: operator (?)

Am I missing something?

**Comment by flan6** [comment]
Can confirm this also happens with `something?.something` syntax. The `?` causes the next token to be unexpected

**Comment by lyc0930** [comment]
Yes, It also can be confirmed that optional chaining `?.` causes an "unexpected punc(.)" error.

**Comment by ribeiroeder** [comment]
Unfortunately this extension is abandoned, I recommend using the updated fork made by [Maingron/VSCodeMinify](https://github.com/Maingron/VSCodeMinify)

With the .vsix file that he made available in the releases you can install locally and finally be able to minify JS with ES6.
